src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, region_silo_model, sane_repr
+
+from .data_condition_group import DataConditionGroup
+
+
+@region_silo_model
+class DataCondition(DefaultFieldsModel):
+    """
+    A data condition is a way to specify a logic condition, if the condition is met, the condition_result is returned.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    __repr__ = sane_repr("type", "condition")
+
+    condition = models.CharField(max_length=200)
+    threshold = models.FloatField(blank=True, null=True)
```

Comment:
An example from the doc that `threshold` could be a string value
```
{
	"threshold": "['tag-a', 'tag-b']",
	"condition": "one_of",
	"condition_result": IssuePriority.LOW
	"type": WorkflowActionTagFilter
}
```

Although I'm not sure if it's up to date, since it doesn't include the `comparison` value either.

Reply:
Yeah - i added `comparison` to be the string representation and `threshold` to be the value. i found that when i was adding it to the organization and i wanted to setup a detector - i found i needed to have a string comparison, but only had an float.

alternative could be that we store this in 1 field, but we'd have to parse that into the different types based on the condition type.

any recommendations? (i'm missing JS where i can just set the type to be what i want 🤣)

Reply:
I think either option is fine. I guess that one field that is json with an enforced schema is more consistent, since you're going to need to use json to encode the comparison field either way. But I don't really have a strong opinion on this.